src/comm/LinkInterface.cc

```diff
@@ -23,28 +25,47 @@ LinkInterface::LinkInterface(SharedLinkConfigurationPtr& config, bool isPX4Flow)
 LinkInterface::~LinkInterface()
 {
     if (_vehicleReferenceCount != 0) {
-        qWarning() << "~LinkInterface still have vehicle references:" << _vehicleReferenceCount;
+        qWarning(LinkInterfaceLog) << "~LinkInterface still have vehicle references:" << _vehicleReferenceCount;
```

Comment:
Doesn't this need to be `qCWarning`? Or does this work as well?

Reply:
There [is a difference between `qC`-prefix (`qCDebug`, `qCWarning`, etc) and `q`-prefix (`qDebug`, `qWarning`, etc) logging functions](https://doc.qt.io/qt-5/qloggingcategory.html#checking-category-configuration). Both "work". The former is somewhat more efficient than the latter. I left the call as-is other than to ensure that it gets put into the correct logging channel. I don't anticipate this code to be performance sensitive but if it were performance sensitive then it should move to `qCWarning`.

Reply:
There [is a difference between `qC`-prefix (`qCDebug`, `qCWarning`, etc) and `q`-prefix (`qDebug`, `qWarning`, etc) logging functions](https://doc.qt.io/qt-5/qloggingcategory.html#checking-category-configuration). Both "work". The former is somewhat more efficient than the latter. I left the call as-is other than to ensure that it gets put into the correct logging channel. I don't anticipate this code to be performance sensitive but if it were performance sensitive then it should move to `qCWarning`.

Reply:
All other places in the code which use categorized debug out use the `qC` variants. Mainly because I thought it was the only way to do it. That said I think it would be best to continue to follow that pattern for consistency.